README.md

```diff
@@ -0,0 +1,413 @@
+
+As did [GitHub remove the problematic "Meritocracy" rug from their office](http://readwrite.com/2014/01/24/github-meritocracy-rug), so now do we literally *pull the rug* from under the tyrannical and Patriarchal feet of Linus Torvalds.  In light of the continued reign of [Linus Torvalds](https://en.wikipedia.org/wiki/Linus_Torvalds) as the immature, unprofessional, quick-to-anger, non-inclusive, white, cisgendered and male project ~~leader~~ dictator of the (in)famous [Linux kernel](http://geekfeminism.wikia.com/wiki/Linux), the Feminist Software Foundation has forked where no feminist has forked and **reclaimed** the software bits to **all people**.
+
+**This is the world's first operating system kernel by FEMINISTS, for FEMINISTS.  Women and gay men with internalised misogyny/homophobia who are here to concern troll and [sealion](http://rationalwiki.org/wiki/Sealioning) are not welcome.**
+
+### Subsidiary but equal repositories
+
+The following repos are part of the wider, curvy **ToleranUX** project.  Contributors are encouraged to also to share their fight in our common, intersectional goal of dismantling Toxic Meritocracy in FOSS software.
+
+* [`ToleranUX-coreutils`, implemented as a diversified fork of BusyBox](https://github.com/rektm8/Tolerant-Utils)
+* [`ProgressiveV``, the overseer of runtime privilege](https://github.com/rektm8/ProgressiveV)
+* [`kprivcheck`, kernel module for privilege checking](https://github.com/saatanaperkele/kprivcheck)
+
+### News and TODO
+
+The specification is almost done.  The parts where there is input still needed are: supported architectures and form factors; inclusion of more `coreutils` to be corrected to feminist thought; and inclusion of more Kernel Daemons under ~~SystemD~~ `SystemV`.
+
+An initial implementation of the `TolernUX-coreutils` is now done in as a ~~`shell .rc`~~ `User-Interactive Free To Be Thee Devices (UIFTBTD) .rc`, which can be [found here](https://github.com/The-Feminist-Software-Foundation/ToleranUX/tree/mistress/ToleranUX-utils).
+
+### How to contribute
+
+Absolutely no coding experience is necessary: all code are equal in the eyes of the Feminist Software Foundation.  There is no objective way to determine whether one person's code is better than another's. In light of this fact, all submitted code will be equally accepted.  However, marginalized groups, such as wom\*n and trans\* will be given priority in order to make up for past discrimination.  Simply submit a pull request for any submission, whether code, artwork, or even irrelevant bits — nothing is irrelevant in the grand struggle for a Truly Tolerant UNIX-ike Kernel!
+
```

Comment:
"No coding experience is necessary." Oh, so now we are submitting crappy code to the OS that, oh, 60% of the world's servers use?